charts/catalog/templates/rbac.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
+kind: List
```

Comment:
it lets you tweak things live, then reexport with a `kubectl get -o yaml` of the objects you're interested in